lib/cluster/s3_distribution_bucket.rb

```diff
@@ -74,7 +73,7 @@ def self.object_keys_to_delete(object)
       end
     end
 
-    def self.default_bucket_policy
+    def self.default_bucket_policy_for(bucket_name)
```

Comment:
That's definitely an option. Can you pull together what this default bucket policy should look like? I think we still don't want to put in the "forbid delete" rules for now.